Makefile

```diff
@@ -103,7 +103,8 @@ COQDEFS := --language=none																\
 	-r '/^[[:space:]]*Tactic[[:space:]]+Notation.*[[:space:]]"\([[:alnum:]'\''_]+\)"[[:space:]]/\1/'										\
 	-r '/^[[:space:]]*Delimit[[:space:]]+Scope[[:space:]]+[[:alnum:]'\''_]+[[:space:]]+with[[:space:]]+\([[:alnum:]'\''_]+\)[[:space:]]*\./\1/'
 
-$(foreach P,$(PACKAGES),$(eval TAGS-$P: $(filter UniMath/$P/%,$(VFILES)); etags -o $$@ $$^))
+$(foreach P,$(PACKAGES),$(eval VFILES_$P := $(filter UniMath/$P/%,$(VFILES))))
```

Comment:
The Makefile already allowed for making not only the file TAGS, but also a tags file for each package, e.g., TAGS-Ktheory.  But the etags command for them was lacking the command line options that allowed it to locate the Coq definitions.  I also made those other TAGS file depend on the Makefile, so they would get remade if the Makefile changed, for example, if the command line options are changed.